src/tests/unit/tests/DetailsView/components/__snapshots__/details-list-issues-view.test.tsx.snap

```diff
@@ -1,78 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`DetailsListIssuesView should return  issues table with violations to null 1`] = `
```

Comment:
Can we change this test case to use getElement as well?

Reply:
for sure